apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/TraceSnippets.cs

```diff
@@ -216,9 +216,9 @@ public void ConfigureServices(IServiceCollection services)
 
             // Register an HttpClient for outgoing requests.
             services.AddHttpClient("tracesOutgoing")
-            // Be sure to add the UnchainedTraceHeaderPropagatingHandler which will guarantee
-            // that the outgoing requests will be traced.
-            .AddHttpMessageHandler<UnchainedTraceHeaderPropagatingHandler>();
+            // The next call guarantees that outgoing requests made with the HTTP Client being configured
+            // will be traced.
```

Comment:
I thought it would only add tracing if trace was enabled for the request being handled? Maybe I'm being confused with something else.

Reply:
No, you are right, changing now.